src/routes/(admin)/tests/test-[type]/+page.svelte

```
@@ -36,6 +36,20 @@
 
 	let currentScreen: number = $state(typeOfScreen.main);
 
+	let filteredTags: String[] = $state([]);
+	let filteredStates: String[] = $state([]);
+
+	let filteredTests = $derived.by(() => {
+		return filteredTags.length === 0 && filteredStates.length === 0
+			? data && data?.tests
+			: data &&
+					data?.tests.filter(
+						(test: any) =>
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)